src/openrct2/network/network.cpp

```diff
@@ -743,6 +743,7 @@ void Network::SetupDefaultGroups()
 	user->ToggleActionPermission(16); // Modify Groups
 	user->ToggleActionPermission(17); // Set Player Group
 	user->ToggleActionPermission(18); // Cheat
+	user->ToggleActionPermission(21); // Modify Tile
```

Comment:
I'm not entirely sure what you mean by having a name.. the actions are based on the string ids, not the actual action index, if that's what you mean.